v1-scripts/prepare-release

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+cfg=$1
+eval $(v1-scripts/configs/build-settings.py "$cfg")
+
+flatname=$(echo "$cfg" | sed 's@/@_@g')
+dirname="$flatname"
+
+mkdir -p .bin
+cp ".pio/build/$env/$out" ".bin/$flatname.$out"
+
+mkdir -p .zip
+ln -sf ".." ".zip/$dirname-src"
+(cd ".zip" && zip -prq "$flatname-src.zip" "$dirname-src"/*)
```

Comment:
Do we need to clean up any build products before zipping?

Reply:
The magic of /* - it will not pick up .pio - the implication is that build artifacts should always be stored in directories that start with a dot.